sdk/modules/ui/cpp/include/metavision/sdk/ui/utils/opengl_api.h

```diff
@@ -14,18 +14,29 @@
 
 #ifdef _USE_OPENGL_ES3_
 #include <GLES3/gl3.h>
-// While we keep support for OpenGL, we need to provide a
-// dummy implementation for Glew init function
-#define GLEW_OK 0
-inline int glewInit(void) {
-    return GLEW_OK;
-}
 #else // OpenGL
+#if defined(__APPLE__) && !defined(__linux__)
+#define GL_SILENCE_DEPRECATION
+#include <OpenGL/gl3.h>
+#else
+#if defined(WIN32)
```

Comment:
Nit: `#elif`

Reply:
I don't believe I can change the logic here. I would end up having to write a separate case for linux and including `#include <GL/glew.h>` and `#include <GL/gl.h>` in both logic branches for WIN32 and linux.

I was able to collapse the first #else to and #elif.